Updates to KFP End-to-end tutorial (Issue #860) (#890)
kubeflow/website: content/docs/pipelines/tutorials/pipelines-tutorial.md

1. Select or create a project on the [GCP Console][gcp-console].
1. Make sure that billing is enabled for your project. See the guide to
  [modifying a project's billing settings][billing-guide].

> 1. Grant users access to kubeflow with the `Owner` and `IAP-secured Web App User` roles:

Reviewer: ```suggestion
1. Grant your team access to Kubeflow by granting them the **Owner** and **IAP-secured Web App User** roles:
```

      ```

>       gcloud projects add-iam-policy-binding <PROJECT>

Reviewer: Please remove this script block. I was wrong, project owner cannot be granted from the command line. They need to use Cloud Console.

Author: I adjusted this, removing the 'Owner' part but leaving the IAP role. Or do they all need to be removed?

      --member user:<EMAIL> --role <ROLE>
      ```

>     For more details, see [Granting, changing, and revoking access to resources][granting-changing-revoking-access].

Reviewer: Suggestion: You can probably streamline this information by focusing on granting access. How about something like "For more details, see the guide to granting access to resources on GCP."

1. (Optional) If you want to examine your cluster while waiting for the UI to
   be available, you can use `kubectl` to connect to your cluster:

>   * Connect your cloud shell session to the cluster:

Reviewer: ```suggestion
  * Connect your Cloud Shell session to the cluster:
```

  environment, run the following commands:   

    ```bash

>     apt-get update; 

Reviewer: ```suggestion
    apt-get update 
```
